src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--mon calendar__month-length--31">
+      <div class="day"></div>
```

Comment:
`day` is related to `calendar` block, so it's better to show this and change class name to `calendar__day` according to BEM

Reply:
I had done like that, but lint said that it was wrong